app/views/layouts/forever_style_guide/application.html.erb

```diff
@@ -14,7 +14,11 @@
   <%= csrf_meta_tags %>
 </head>
 <body class="style-guide" data-spy="scroll" data-target=".style-guide-scrollspy">
-  <%= render partial: "layouts/forever_style_guide/navigation" %>
+
+  <% if !content_for?(:hide_nav) %>
```

Comment:
Yeah it would go into https://github.com/forever-inc/forever-style-guide/blob/master/app/controllers/forever_style_guide/style_controller.rb#L11

This actually loads for all demo pages anyway, so considering we want to almost always override the header here anyway this should fix that annoying double dropshadow thing there as well.